modules/ROOT/pages/provisioning-vmware.adoc

```diff
@@ -155,3 +156,31 @@ VM_NAME='fcos-node01'
 govc device.serial.add -vm "${VM_NAME}"
 govc device.serial.connect -vm "${VM_NAME}" "[datastore] ${VM_NAME}/console.log"
 ----
+
+== Modifying OVA metadata
+
+NOTE: While we provide these instructions for modifying the OVA, we cannot
+guarantee that any modifications to the OVA metadata will result in a usable
+guest VM.
+
+Fedora CoreOS ships the VMware OVA with certain metadata defaults in the OVA file.
+The hardware version and default firmware are of particular interest, as they
+can affect how the guest VM operates in the hypervisor. These defaults may
+prevent the use of FCOS as a guest VM in older versions of VMware products and
+it may be necessary to modify the metadata in order to use FCOS as a guest VM.
+
+The VMware OVA is simply a tarball that contains the files `disk.vmdk` and
+`coreos.ovf`. In order to edit the metadata used by FCOS as a guest VM, you
+should untar the OVA artifact, edit the OVF file, then create a new OVA file.
+
+In the example below, the hardware version in the OVF is changed from the
+default value of `17` to a newer version of `13`. (Note: the defaults in the
```

Comment:
Did 13 and 17 get swapped here?

Reply:
Oops, good catch.

Reply:
Actually, I am just going to update the wording to align with the example of changing from a newer default value to an older value.